s4cmd.py

```diff
@@ -830,6 +830,16 @@ def update_privilege(self, obj, target):
       os.chmod(target, int(obj['Metadata']['privilege'], 8))
 
   @log_calls
+  def print_files(self, source):
+    '''Print out a series of files'''
+    sources = self.source_expand(source)
+
+    for source in sources:
+      s3url = S3URL(source)
+      response = self.s3.get_object(Bucket=s3url.bucket, Key=s3url.path)
+      message(response['Body'].read())
```

Comment:
Good catch! Fixed that. Works for arbitrarily-many `%` characters now, and yes, Unicode just always worked:

![s4cmd](https://cloud.githubusercontent.com/assets/14872/16732790/d4f375fa-474c-11e6-8b30-21e6340f8bdd.png)